system/repositories.rs

```diff
@@ -43,11 +43,9 @@ pub mod user_repository {
         let (tx, result) =
             execute_read_pipeline(tx, &query.into_pipeline());
         let rows = result.unwrap();
-        let mut users: Vec<User> = vec![];
-        for row in rows {
-            let name  = get_string(&tx, &row, "n");
-            users.push(User::new(name))
-        }
+        let users = rows.iter().map(|row| {
+            User::new(get_string(&tx, &row, "n"))
+        }).collect();
         users
```

Comment:
Can be just
```
rows.iter().map(|row| User::new(get_string(&tx, &row, "n"))).collect()
```
(without `let`)

Reply:
(not going to push you to do so, it's just cleaner. If we want to highlight what this collection means, it should be fixed in the method's name)

Reply:
Fixed in 12cc62ffeacb542cb8e6316e14a0d05c41578431.